.github/workflows/crystal.yml

```
@@ -25,16 +25,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # Available images https://github.com/actions/runner-images?tab=readme-ov-file#available-images
         include:
           - os: ubuntu-latest
             crystal: latest
           - os: windows-latest
             crystal: latest
-          - os: macos-latest
+          - os: macos-15-large # Intel platform
+            crystal: latest
+          - os: macos-latest-xlarge # ARM platform
```

```suggestion
          - os: macos-13 # Intel platform
            crystal: latest
          - os: macos-latest # ARM platform
```

Going based off these docs https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories 

It looks like having `-large` or `-xlarge` require payment on file in order to run. Also, it seems for the free tier intel you have to go back to `-13`? Unless I'm reading this wrong...

---

Yep, It should work.

---

## Tophat

It seems the fix is working.

macos-latest [log](https://github.com/crystal-loot/webdrivers.cr/actions/runs/13932099497/job/38991579764#step:4:21):

```
LLVM: 15.0.7
Default target: aarch64-apple-macosx11.0
```

macos-13 [log](https://github.com/crystal-loot/webdrivers.cr/actions/runs/13932099497/job/38991579766#step:4:21):

```
LLVM: 15.0.7
Default target: x86_64-apple-macosx11.0
```